rpc/klay_get_block_receipts/boilerplate.js

```diff
@@ -0,0 +1,64 @@
+const path = require('path')
+const dotenv = require('dotenv')
+const Caver = require('caver-js')
+const ROOT_DIR = path.join(__dirname, '../..'); // Path can be changed based on its actual location.
+
+// You can directly input values for the variables below, or you can enter values in the caver-js-examples/.env file.
+let nodeApiUrl = ""; // e.g. "https://node-api.klaytnapi.com/v1/klaytn";
+let accessKeyId = ""; // e.g. "KASK1LVNO498YT6KJQFUPY8S";
+let secretAccessKey = ""; // e.g. "aP/reVYHXqjw3EtQrMuJP4A3/hOb69TjnBT3ePKG";
+let chainId = ""; // e.g. "1001" or "8217";
+let senderAddress = ""; // e.g. "0xeb709d59954f4cdc6b6f3bfcd8d531887b7bd199"
+let senderPrivateKey = ""; // e.g. "0x39a6375b608c2572fadb2ed9fd78c5c456ca3aa860c43192ad910c3269727fc1"
+
+/**
+ * Boilerplate code about "How to get receipts included in a block."
+ * Related reference - Korean: https://ko.docs.klaytn.com/bapp/json-rpc/api-references/klay/block#klay_getblockreceipts
+ * Related reference - English: https://docs.klaytn.com/bapp/json-rpc/api-references/klay/block#klay_getblockreceipts
+ */
+async function main() {
+    try {
+        loadEnv()
+        run()
+    } catch (err) {
+        console.error(err)
+    }
+}
+
+main()
+
+function loadEnv() {
+    result = dotenv.config({path: `${ROOT_DIR}/.env`})
```

Comment:
Can we change this variable name from result to something else?

Reply:
@jimni1222 
How about `envs`? 😃 
That will be more meaningful than result.

Reply:
I added suggestion with `envs`.